src/main/java/org/jabref/logic/exporter/XmpExporter.java

```diff
@@ -64,8 +67,16 @@ public void export(BibDatabaseContext databaseContext, Path file, Charset encodi
     }
 
     private void writeBibToXmp(Path file, List<BibEntry> entries, Charset encoding) throws IOException {
+        String xmpContent = XmpUtilWriter.generateXmpString(entries, this.xmpPreferences);
```

Comment:
Can we add a second method there? `generateXmpStringWithoutXmpDeclaration`? Maybe rename to current functio to `generateXmpStringWithXmpDeclaration`. The term "XmpDeclaration" stems from "XmlDeclaration". See https://wiki.selfhtml.org/wiki/XML/Regeln/XML-Deklaration.